cmd/metadata/config/config.go

```diff
@@ -23,6 +23,7 @@ func (c *Config) Substitute() error {
 			return err
 		}
 	}
+	c.Config.Hasura.SetSourceName()
```

Comment:
`c.Config.Hasura` is a pointer and may be nil. It will be panic here if config does not contain `hasura` section.

Reply:
Yes, I saw your change in go-lib. Going to change.

I had not considered that it's a pointer. Still new to GO (and have maybe done a bit too much js recently :) ).